lms/djangoapps/grades/tasks.py

```diff
@@ -118,9 +119,10 @@ def _recalculate_subsection_grade(task_func, **kwargs):
 
     except Exception as exc:   # pylint: disable=broad-except
         if not isinstance(exc, KNOWN_RETRY_ERRORS):
-            log.info("tnl-6244 grades unexpected failure: {}. kwargs={}".format(
+            log.info("tnl-6244 grades unexpected failure: {}. task id: {}. kwargs={}".format(
                 repr(exc),
-                kwargs
+                recalculate_subsection_grade_v3.request.id,
```

Comment:
Change this to `task_func.request.id`?  So it works with whatever task was actually called.
Also, shouldn't it be `self.request`?